docs/capabilities.md

```diff
@@ -13,6 +13,7 @@ Use Tableau capabilities to customize and tune your connector behavior.
 * [Stored Procedures](#stored-procedures)
 * [Isolation Level](#isolation-level)
 * [Uncommon](#uncommon)
+* [OAuth](#oauth)
```

Comment:
Can you link this from the mcd page instead of the here.  I think it makes more sense that way because its used first within the context.  I also want to avoid partners accidently thinking they'd put the OAuth related capabilities into the manifest.xml